Lib/logging/__init__.py

```diff
@@ -128,6 +128,9 @@ def getLevelName(level):
     If a numeric value corresponding to one of the defined levels is passed
     in, the corresponding string representation is returned.
 
+    If a string representation of the level is passed in, the corresponding
+    numeric value is returned.
+
     Otherwise, the string "Level %s" % level is returned.
```

Comment:
Suggest changing this to "If no matching numeric or string value is passed inm the string 'Level %s' % level is returned."